configs.py

```diff
@@ -36,7 +38,7 @@ class WorkloadConfig(Config):
     duration: float
 
     # Number of tokens in the context (approximate number)
-    context_length: int
+    context_length: Union[int, List[int]]
```

Comment:
What does it mean if we pass a list to the context length? What if we want to generate 10 requests but the list only has 5 elements?

Reply:
Most tests still use the dummy workload, which means you only need to assign a single integer for `context_length`. However, for the test, `test_vary_length_workload`, you should specify as many `context_length` list elements as there are request numbers. I'll stress this in comment of `test_vary_length_workload`.